09-secret-management-and-ingress-controller.md

```diff
@@ -48,7 +48,7 @@ Previously you have configured [workload prerequisites](./08-workload-prerequisi
              objectName: traefik-ingress-internal-aks-ingress-tls
              objectAlias: tls.key
              objectType: secret
-       tenantId: $TENANTID_AZURERBAC_AKS_BASELINE
+       tenantID: $TENANTID_AZURERBAC_AKS_BASELINE
```

Comment:
I know this is pretty much intended as this is documented from the PR body. But just out of curiosity, is this a bug fix?

Reply:
yeah, very good idea to update this one based on this.